RELEASES.md

```diff
@@ -232,6 +237,21 @@ completed, open a PR using the process above.
 Only when the bug is not seen in main and must be made for the specific
 release branch should you open a PR with new code.
 
+### Upgrade Path
+
+The upgrade path for containerd is such that the 0.0.x patch releases are
+always backward compatible with its major and minor version. Minor (0.x.0)
+version will always be compatible with the previous minor release. i.e. 1.2.0
+is backwards compatible with 1.1.0 and 1.1.0 is compatible with 1.0.0. There is
+no compatibility guarantees for upgrades that span multiple, _minor_ releases.
+For example, 1.0.0 to 1.2.0 is not supported. One should first upgrade to 1.1,
+then 1.2.
+
+There are no compatibility guarantees with upgrades to _major_ versions. For
+example, upgrading from 1.0.0 to 2.0.0 may require resources to migrated or
```

Comment:
What about upgrading from 1.6 to 2.0 ?

Reply:
I cleaned up this section some. Maybe I should point to somewhere else or mention something else about the warnings? @samuelkarp

Reply:
Commented below, but 1.6 -> 2.0 is probably not a thing we will want to support once 1.7 is LTS.  Users can upgrade 1.6 (LTS) -> 1.7 (LTS) -> 2.0 (Active).  We might revisit this if 2.0 does not become LTS and we want to say what else users should do.